### ISSUE TITLE
Can't load transitive dependencies

### ISSUE THREAD

**Issue by callumlocke** [mjeanroy/rollup-plugin-bower-resolve#4]
Thanks for writing this plugin! One problem I've found...

My project has a bower.json that depends on `module-a` and I run `bower install`. It turns out that `module-a` also has its own bower.json, and it depends on `module-b`, so my `bower_components` now contains both modules, because bower automatically gets the whole tree of deps for me and flattens it.

The problem is, this Rollup plugin acts as if module-b is not installed, so I get an error from rollup, something like:

```
'module-b' is imported by bower_components/module-a/main.js, but could not be resolved – treating it as an external dependency
```

...even though I can confirm that `bower_components/module-b/` exists.

As a workaround, I can add module-b to my own bower.json, and this works. But ideally this wouldn't be necessary.

**Comment by mjeanroy** [comment]
@callumlocke Thanks for for reporting this! Let me check it, I'll release a patch when fixed ;)

**Comment by mjeanroy** [comment]
@callumlocke It should be fixed with `0.4.0`.
